content/docs/for-developers/sending-email/building-an-x-smtpapi-header.md

```diff
@@ -76,6 +79,88 @@ Schedule your email send time using the `send_at` parameter within your X-SMTPAP
+RCPT TO:person2@sendgrid.com
+250 Recipient address accepted
+DATA
+354 Continue
+From: "Tira Misu" <tiramisu@example.com>
+To: <person1@sendgrid.com>
+Subject: Test message subject
+This is the test message body.
+.
+250 Ok: queued as Yo60h6C5ScGPeP5fUWU3Kw
+```
+
+Notice that there are two recipients designated by RCPT commands, "person1@sendgrid.com" and "person2@sendgrid.com." These addresses are both part of the message envelope.
+
+In the DATA command, there is also a "To" header. This header lists only "person1@sendgrid.com." This is where differentiating between the envelop "To" (RCPT TO) and header "To" becomes important. The header "To" set in the DATA command does not tell the sending email server to deliver the message to any address, only the RCPT or envelop "To" does this. The header "To" instead provides recipients with a friendly display of any addresses included in the "To" header.
+
+To achieve BCC behavior, you can deliver a message to a recipient by adding them in a RCPT but omit their address from the header "To". The message will be delivered to each RCPT address, but only the addresses listed in the header "To" will be visible to other recipients.
+
+In the previous code sample, "person2@sendgrid.com" will be treated like a BCC address because that address is not included in the header "To." The "person1@sendgrid.com" is listed in the header "To" and will therefore be visible to other recipients of the message. This means both recipients receive the message, but recipient2@sendgrid.com is not visible to recipient1@sendgrid.com while recipient1@sendgrid.com is visible to recipient2@sendgrid.com.
+
+By omitting any addresses from the header "To," you will be hiding them from the other recipients and therefore treating them like BCC recipients.
+
+#### BCC with the X-SMTPAPI
+
+The following code shows an example of an SMTP transaction with BCC behavior using the X-SMTPAPI header. How this example achieves BCC behavior is explained following the example.
+
+```shell
+235 Authentication successful
+MAIL FROM:tiramisu@example.com
+250 Sender address accepted
+RCPT TO:sender@sendgrid.com
+250 Recipient address accepted
+DATA
+354 Continue
+x-smtpapi: {"to":["person1@sendgrid.com","person2@sendgrid.com"]}
+From: "Tira Misu" <tiramisu@example.com>
+Subject: This is a test
+Body of test message.
+.
+250 Ok: queued as uKIPst3rQtCl_hLVB9RvEw
+```
+
+Using the X-SMTPAPI header achieves BCC behavior in a slightly different way than by omitting addresses from a "To" header. The X-SMTPAPI header's "to" field allows you to set multiple recipients in a JSON array.
+
+```json
+{
+  "to": ["person1@sendgrid.com", "person2@sendgrid.com"]
+}
+```
+
+When SendGrid receives the message and parses the X-SMTPAPI header, it will convert each address in the X-SMTPAPI "to" field to a separate RCPT TO or _envelope_ address. The X-SMTPAPI "to" field in the previous code sample will eventually look something like the following example.
```

Comment:
This section invites some unnecessary questions when I read it, but let me know if what I think would make more sense actually does...
```
When SendGrid receives the message and parses the X-SMTPAPI header, it will treat each recipient address in the X-SMTPAPI "to" field as a separate RCTP TO or _envelope_ address. This means each recipient will receive the same DATA content, but with an added friendly display "to" header set to its own address.
```
Then maybe the example below for one of the recipients?

Reply:
This is great! Thank you!